### ISSUE TITLE
torch.load import error

### ISSUE THREAD

**Issue by Guoxs** [abhi4ssj/relaynet_pytorch#4]
I tried to follow the example code that was given, it gives the following error in Ubuntu 18.04, any help would be appreciated.
`ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-50-4839094a99d3> in <module>()
      4 from torch.autograd import Variable
      5 
----> 6 relaynet_model =  torch.load('models/relaynet_good.model')

~/.local/lib/python3.6/site-packages/torch/serialization.py in load(f, map_location, pickle_module)
    301         f = open(f, 'rb')
    302     try:
--> 303         return _load(f, map_location, pickle_module)
    304     finally:
    305         if new_fd:

~/.local/lib/python3.6/site-packages/torch/serialization.py in _load(f, map_location, pickle_module)
    467     unpickler = pickle_module.Unpickler(f)
    468     unpickler.persistent_load = persistent_load
--> 469     result = unpickler.load()
    470 
    471     deserialized_storage_keys = pickle_module.load(f)

ModuleNotFoundError: No module named 'networks.classifiers'
`

**Comment by Guoxs** [comment]
Hi, I checked the API and it seems like Pytorch requires 'model.py' in the folder you use to load the trained models, but I haven't seen any file named 'model.py' in this repository, and I tried to write a 'model.py' file according to your code but it didn't work. The API recommends saving only the state dicts of the model and not whole model objects. I wonder if you are convenient to provide state dicts of trained model, thank you so much!
